src/OrbitGl/GraphTrack.cpp

```diff
@@ -102,11 +102,6 @@ void GraphTrack<Dimension>::UpdatePrimitives(Batcher* batcher, uint64_t min_tick
   DrawSeries(batcher, min_tick, max_tick, graph_z);
 }
 
-template <size_t Dimension>
-void GraphTrack<Dimension>::OnTimer(const orbit_client_protos::TimerInfo& timer_info) {
-  track_data_->AddTimer(/*depth=*/0, timer_info);
```

Comment:
I supposed that we were using OnTimer in GraphTrack

Reply:
Suppose to be fine as `OnTimer` methods are override in the derived classes.

But I'm curious how the graph track related timers are stored in the capture file now.

Reply:
we do not store TimerInfos in capture file anymore - only the raw data coming from the service

Reply:
so all timer infos UI sees are calculated when orbit loads raw data from the file.